core/src/utils/native/haptic.ts

```diff
@@ -68,7 +131,7 @@ const HapticEngine = {
     if (this.isCapacitor()) {
       engine.selectionStart();
     } else {
-      engine.gestureSelectionStart();
+      (engine as unknown as TapticEngine).gestureSelectionStart();
```

Comment:
Why do we need to cast as `unknown` first?

Reply:
When converting between an explicit type signature to another type signature that doesn't overlap, you need to cast to `unknown` which is the equivalent of "I know what I am doing" to disable Typescript errors.

In this case, the `engine` is Cordova's taptic engine, but the return type of `getEngine` is `HapticsEngine`. This strategy was used to get explicit type checking for the Cordova usage.

Reply:
Thanks! Will this no longer be needed once Cordova support for the haptic plugin is dropped?

Reply:
Correct! We can remove this fallback and the `TapticEngine` type once the Cordova support is removed.